### DISCUSSION TITLE
how to use block afterJavaScriptEvaluated

### DISCUSSION THREAD

**Discussion by MahaSabri1981** [vknow360/CustomWebView#6] [General]
how to use block afterJavaScriptEvaluated
![Capture](https://user-images.githubusercontent.com/78137256/106865543-ff29e380-66d3-11eb-87ec-6d93036c5d71.PNG)

**Comment by vknow360** [comment]
Hi @MahaSabri1981 
You can evaluate java script using 'EvaluateJavaScript' block and get the result using that event.
